src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+@use '../utils/variables' as *;
+
+.card {
+  box-sizing: border-box;
+  padding: 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  &__name {
+    font-size: 1.2em;
+    margin-top: 40px;
+    line-height: 18px;
+    font-weight: $secondary-weight;
```

Comment:
The `font-weight` property is defined here as `$secondary-weight`. Ensure this is the intended value and remove any redundant declarations.